runtime/service.go

```diff
@@ -251,17 +268,18 @@ func (s *service) proxyStdio(ctx context.Context, stdin, stdout, stderr string,
 
 func proxyIO(ctx context.Context, path string, cid, port uint32, in bool) {
 	if path == "" {
+		log.G(ctx).WithField("cid", cid).WithField("port", port).Warn("skipping IO, path is empty")
 		return
 	}
-	log.G(ctx).Debug("setting up IO for " + path)
+	log.G(ctx).WithField("path", path).WithField("cid", cid).WithField("port", port).Debug("setting up IO")
 	f, err := fifo.OpenFifo(ctx, path, syscall.O_RDWR|syscall.O_NONBLOCK, 0700)
 	if err != nil {
-		log.G(ctx).WithError(err).Error("error opening fifo")
+		log.G(ctx).WithError(err).WithField("path", path).Error("error opening fifo")
 		return
 	}
 	conn, err := vsock.Dial(cid, port)
 	if err != nil {
-		log.G(ctx).WithError(err).Error("unable to dial agent vsock")
+		log.G(ctx).WithError(err).WithField("path", path).Error("unable to dial agent vsock")
```

Comment:
nit: could include something like `unable to dial agent vsock IO port` to distinguish from the other vsock dial we do

Reply:
Done